sdk/scheduler/src/test/java/com/mesosphere/sdk/offer/OfferUtilsTest.java

```diff
@@ -82,6 +85,22 @@ public void testDeclineOffers() {
         verify(mockSchedulerDriver).declineOffer(eq(offerIds.get(1)), any());
     }
 
+    @Test
+    public void testUnreservedResourcesIsProcessable() {
+        Protos.Resource resource = ResourceTestUtils.getUnreservedCpus(1.0);
+        Assert.assertTrue(ResourceUtils.isProcessable(resource, null));
+    }
+
+    @Test
+    public void testReservedByThisFrameworkIsProcessable() {
```

Comment:
Test for alien resources aren't processable?

Reply:
https://github.com/mesosphere/dcos-commons/blob/master/sdk/scheduler/src/test/java/com/mesosphere/sdk/offer/ResourceUtilsTest.java#L18-L120